src/helpers/queryDataParser.js

```diff
@@ -0,0 +1,151 @@
+import _filter from 'lodash/filter';
+import { texts } from '../config';
+
+import { QUERY_TYPES } from '../queries';
+import { eventDate } from './dateTimeHelper';
+import { mainImageOfMediaContents } from './imageHelper';
+import { momentFormat } from './momentHelper';
+import { shareMessage } from './shareHelper';
+import { subtitle } from './textHelper';
+
+export const parseEventRecords = (data, skipLastDivider = false) => {
+  return data?.map((eventRecord, index) => ({
+    id: eventRecord.id,
+    subtitle: subtitle(
+      eventDate(eventRecord.listDate),
+      eventRecord.addresses?.[0]?.addition || eventRecord.addresses?.[0]?.city
```

Comment:
it wasn't easy to catch the change to optional chaining here from the diffs. refactoring this is missing in the commit message, which should have been pointed out there to avoid problems.

Reply:
yes it is simply making sure that we do not do `undefined[0]` by optional chaining.